jobserver/models/outputs.py

```diff
@@ -29,7 +29,7 @@ class Decisions(models.TextChoices):
     )
     snapshot = models.ForeignKey(
         "Snapshot",
-        on_delete=models.SET_NULL,
+        on_delete=models.CASCADE,
```

Comment:
Would we delete a snapshot if the wrong file was requested or published? Having the audit trail of who made the request and what decision was made sounds like it'll be important even if the file is no longer there.

Reply:
Thank you. That makes sense. It sounds like we'll have an audit trail if files are deleted, which is my only concern. I don't think we'll need that extra step either, so I think CASCADE is the right choice here.